aqua/qt53/Portfile

```diff
@@ -0,0 +1,1775 @@
+
+            # https://codereview.qt-project.org/#/c/140876/
+            # this qt55 patch does not apply on qt53 and does not appear to be needed
+            # patchfiles-append patch-stdloc.diff
+
+            #https://codereview.qt-project.org/#/c/138349
+            # this qt55 patch has been modified for qt53
+            patchfiles-append patch-rpath.diff
+
+            # as of 5.6.1, avoid "... is lacking an install target"
+            # see https://codereview.qt-project.org/#/c/156610/1
+            patchfiles-append patch-install_examples.diff
+
+            # see https://stackoverflow.com/questions/14506151/invalid-symbol-redefinition-in-inline-asm-on-llvm
+            # only runs test code on 32-bit systems
+            patchfiles-append patch-tst_benchlibcallgrind.diff
+
+            # this test might still cause problems, but with 5.4.2->5.5.0, it is no longer run
+            # tests/auto/corelib/plugin/qpluginloader/qpluginloader.pro:
+            #     -macx-*: SUBDIRS += machtest
+            #     +macx-*: contains(QT_CONFIG, private_tests): SUBDIRS += machtest
+            #
+            # When testing, ensure that a universal object file is not inadvertently created.
+            # this qt55 patch does not apply on qt53 and does not appear to be needed
+            # patchfiles-append patch-machtest.diff
+
+            # see #52200
+            # see https://codereview.qt-project.org/#/c/164673/
+            patchfiles-append patch-xcrun.diff
+
+            # see https://bugreports.qt.io/browse/QTBUG-62266
+            patchfiles-append patch-sdk10_13.diff
+
+            # avoid non-standard C++ that Xcode Clang accepts but other compilers do not
+            # see https://lists.llvm.org/pipermail/llvm-bugs/2017-January/053481.html
+            # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=11764
+            patchfiles-append patch-qualified_reference.diff
+
+            # find the Rez program
+            patchfiles-append patch-find_rez.diff
+            post-patch {
+                reinplace \
+                    "s|__MACPORTS_Rez__|[exec xcrun --find Rez]|g" \
+                    mkspecs/common/mac.conf \
+                    mkspecs/features/mac/rez.prf
+            }
+            
+            # new qt53 patch - don't force deployment target to 10.7 for c++11
+            patchfiles-append patch-allow-c++11-with-106OSDK.diff
+            
+                        
+            # in /Developer/SDKs/MacOSX10.6.sdk/usr/include/rpcsvc/yp_prot.h we have the following block,
+            # but c++11 already defines bool, so an error occurs 
+
+#           #ifndef BOOL_DEFINED
+#           typedef unsigned int bool;
+#           #define BOOL_DEFINED
+#           #endif
+
+            # defining BOOL_DEFINED for the nis test fixes this for qtbase build. May need it later
+            # and if so will have to add it to the qmake5 PG
+            patchfiles-append patch-qt53-SL-nispro.diff
```

Comment:
I believe `configure.args-append {-D BOOL_DEFINED}` would prevent the need for another patch file.

Reply:
I thought of that - but I wondered where else that define might go and what it might do as it would have spread through the whole qt53-qtbase build. 

Also, over the years I have found that adding single defines like that to a cmake build is difficult (you almost always have to patch CMakeLists.txt to get them to stick), and I figured qmake would be similar -- but I have not come across that syntax you are showing there for a configure.arg-append....